cellxgene_schema_cli/tests/fixtures/examples_validate.py

```diff
@@ -145,7 +145,7 @@
 var_expected = pd.DataFrame(
     [
         ["spike-in", False, "ERCC-00002 (spike-in control)", "NCBITaxon:32630", 0],
-        ["gene", False, "MACF1", "NCBITaxon:9606", 42738],
+        ["gene", False, "MACF1", "NCBITaxon:9606", 70573],
```

Comment:
is this the length? Trying to remind myself of the column headers which are not included in the csv files themselves...

Reply:
Ah yes, per the edit you made below...

Reply:
That's a little odd though. The gene name is not changing... but the length is...? 🤔

Reply:
i asked here: https://czi-sci.slack.com/archives/C06AVGFV222/p1707412552115149 and we'll see what they say. i don't entirely understand it but i'd be surprised if we couldn't just trust the data in the ontology files... i think that i'll just merge this PR into `joyce/ont-gencode-bump` for now and we can see if there's any additional changes to be made